similar_dev_search/cli/fetch_repos.py

```diff
@@ -0,0 +1,67 @@
+from pathlib import Path
+
+import click
+from joblib import Parallel, delayed
+
+from similar_dev_search.constants import JSONS_PATH, N_JOBS, REPOS_PATH
+from similar_dev_search.services.file_system import FileSystemService, JsonService
+from similar_dev_search.services.git import GitService, GithubService
+from similar_dev_search.services.git import RepositoryProvider
+from similar_dev_search.setup import setup
+
+
+@click.command()
+@click.option('--username', default='scikit-learn', help='Username for start repo')
+@click.option('--reponame', default='scikit-learn', help='Start repo name')
+@click.option('--max_repos', default=3, help='Max count of repos')
+def fetch_repos(username: str, reponame: str, max_repos: int) -> None:
+    print("Searching repositories...")
+    repos_list = list(find_repos(username, reponame))
+    print("Searching done...")
+    repositories_count = FileSystemService.get_file_count(JSONS_PATH)
+    Parallel(n_jobs=N_JOBS)(
+        delayed(handle_repo)(repo.owner.login, repo.name)
+        for repo in repos_list[:max(0, max_repos - repositories_count + 1)]
+    )
+
+
+def find_repos(username, reponame, depth=2):
+    ghs = GithubService()
+    repo = ghs.get_repo(username, reponame)
+    contributors = repo.get_contributors()
+    result = {repo}
+    if depth != 2:
+        for contributor in contributors[:5]:
+            stars = ghs.get_user(contributor.login).get_starred()
+            stars = get_top_starred_repos(stars)[:10]
+            for star in stars:
+                result = result | find_repos(star.owner.login, star.name, depth + 1)
+    return result
+
+
+def handle_repo(username: str, reponame: str):
+    """
+    Clone repo and export to json.
+
+    :param username: Username of the repo author.
+    :param reponame: Name of the repo.
+    """
+    ghs = GithubService()
+    gs = GitService()
+    setup(gs)
+    repo = ghs.get_repo(username, reponame)
+    gs.clone_repo(repo.clone_url, REPOS_PATH + repo.name)
```

Comment:
move all hardcoded paths/values to arguments of the function with some default values. So you can easily change them through CLI arguments